src/main/kotlin/no/nav/tilleggsstonader/sak/brev/brevmottaker/BrevmottakereService.kt

```diff
@@ -75,6 +74,7 @@ class BrevmottakereService(
     private fun lagreNyBrevmottakerPerson(behandlingId: UUID, it: BrevmottakerPersonDto) {
         brevmottakereRepository.insert(
             Brevmottaker(
+                id = it.id,
```

Comment:
`lagreNyOrganisasjonsmottaker` trenger også at id'et lagres ned :) 

Hadde kanskje vært fint med noen tester for denne servicen og 👀 Ser det ikke finnes noen

Reply:
Har ikke fått skrevet noen tester her enda.

Reply:
Edit: Jobber med saken! 😁 👇